text_recognizer/data/iam.py

```diff
@@ -37,91 +40,108 @@ class IAM:
     def __init__(self):
         self.metadata = toml.load(METADATA_FILENAME)
 
-    def prepare_data(self) -> None:
+    def prepare_data(self):
```

Comment:
won't `mypy` cry?

Reply:
there's a "no more tears" setting for returning nones that we've turned on